src/chainparams.h

```diff
@@ -70,6 +70,8 @@ class CChainParams
     };
 
     const Consensus::Params &GetConsensus() const { return consensus; }
+    /** Modifiable consensus parameters added by bip135 */
```

Comment:
add "this can only be used during initialization, since modification of the consensus params is not thread safe" to the comment.